docs/design/networking/01-fix-hostname-usage-related-issues.md

```diff
@@ -0,0 +1,350 @@
+# Fix hostname usage related issues
+
+### Table of Contents
+
++ [Background](#background)
+    - [Description](#description)
+    - [Terminology](#terminology)
+    - [Actors and Scenarios](#actors-and-scenarios)
++ [Functional Design](#functional-design)
++ [User Interaction](#user-interaction)
+    - [API design and/or Prototypes](#api-design-andor-prototypes)
++ [Technical Design](#technical-design)
+    - [Notes/Questions/Issues](#notesquestionsissues)
++ [Testing Criteria](#testing-criteria)
+
+
+|||
```

Comment:
The table formatting doesn't seem correct - check the https://github.com/hazelcast/hazelcast/blob/6efc1fe999f35e2a1a1b0400ebb8876dbb243c46/docs/design/networking/01-fix-hostname-usage-related-issues.md